### DISCUSSION TITLE
Clarification about logging limit

### DISCUSSION THREAD

**Discussion by nbelles** [bitcraze/discussions#344] [Q&A]
Hello,

I wanted to clarify the total amount of data that can be logged. Is it possible to have like 10 or 15 variables logged simultaneously? I noticed in the examples (`crazyflie-lib-python/examples/logging/basiclog.py`) that they only use 7 variables and in the [documentation](https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/python_api/) it explains that a log packet (is there a limit to the number of log packets?) allows 7 variables. Having the ability to add multiple logging objects to the Crazyflie makes it seem like you can create multiple logging packets but I have no clue.

Thanks!

**Comment by knmcguire** [comment]
Hi! The packets have a limit of 26 bytes. So you need to know the size of the variables (32 bit float, 16 bit unsigned integer) and from there you can know how much a packet can carry at the time.

Easiest to check this is to use the logging setting functionality of the cfclient: https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#logging, as that will indicate how full your packet is once you select the logging variables
